integration_tests/test_household_structure.py

```diff
@@ -29,3 +30,18 @@ def test_all_households_have_reference_person(tracked_live_populations, time_ste
     # Assert these two sets are identical
     assert non_gq_household_ids.size == reference_person_household_ids.size
     assert np.setxor1d(non_gq_household_ids, reference_person_household_ids).size == 0
+
+
+def test_household_id_and_address_id_correspond(tracked_live_populations):
+    for pop in tracked_live_populations:
+        assert pop["household_id"].notnull().all()
+        assert pop["address_id"].notnull().all()
+        # 1-to-1 at any given point in time
+        assert (pop.groupby("household_id")["address_id"].nunique() == 1).all()
+        assert (pop.groupby("address_id")["household_id"].nunique() == 1).all()
+
+    # Even over time, there is only 1 household_id for each address_id -- they are not reused
+    # Note, however, that the reverse is not true: a household_id can span multiple address_ids
```

Comment:
So that I understand - you're saying that "a household_id can span multiple address_ids" _in different time steps_?

Reply:
Yes. I will clarify this comment.